nixos/modules/config/qt5.nix

```diff
@@ -0,0 +1,108 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.qt5;
+
+  platformThemePackage = if cfg.platformTheme == "gnome"
+    then pkgs.qgnomeplatform
+    else pkgs.libsForQt5.qtstyleplugins;
+
+  stylePackage = if cfg.style == "adwaita"
```

Comment:
How do these two options interact? Are there combinations of settings that aren't valid? Like maybe `platformTheme = "gnome"` and `style = "gtk"`?

Reply:
Yeah, `platformTheme = "gnome"` and `style = "gtk2"` would result in a segmentation fault.

Reply:
Would be good to guard against this somehow then